### PR TITLE
Js-Milkyday-bar

### PR DESCRIPTION
**Pull request by rahalrazika**
- Built with  Webpack
- Created all modules 
- for more details please see the readme file.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    if (event.target.id === 'tab-link-0') navigator(homeComp);
+    if (event.target.id === 'tab-link-1') navigator(aboutComp);
+    if (event.target.id === 'tab-link-2') navigator(menuComp);
+    if (event.target.id === 'tab-link-3') navigator(contactComp);
+  });
+});
+/* content.appendChild(containre);
+content.appendChild(about);
+content.appendChild(menu);
+content.appendChild(contact); */
```
Comment (whiz25) [inline src/index.js]: - remove this comment and any other comments in the code base

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work. The design looks good especially the logo. The app is well structured with many modules.
Let's make some changes before merging this PR.
- [x] add a title to this PR. Learn how [here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request)
- [x] Remove all comments from your codebase


### Martin Afani

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @rahalrazika 
### You have done well :clap: 
- [x] Follow the gitflow outlines, make a development branch main, and create all feature branches to the development. Main should only be for deployment.
### Make the change and submit for another code review.
- Good luck.

**Review by Dandush03 — APPROVED** [review]
# Project Approved
# Great Work :muscle: 
# Happy Coding :nerd_face:

**Pull request merged**